webapp/src/types/common/store.d.ts

```diff
@@ -15,6 +15,7 @@ type GlobalModalState = {
 type GlobalModalActionPayload = {
     isVisible: boolean
     commandArgs: Array<string>
+    args?: Array<string>
```

Comment:
What's the main difference between commandArgs and args here?

Reply:
`commandArgs` will come from slash command and args will come from anywhere like dispatching action on button click, etc